service/src/main/resources/application.properties

```diff
@@ -27,6 +28,11 @@ hive.metastore.uris=thrift://localhost:9083
 hive.metastore.dbname=default
 hive.hmshandler.retry.attempts=15
 hive.hmshandler.retry.interval=2000ms
+
+#Hive jdbc
+hive.jdbc.className=org.apache.hive.jdbc.HiveDriver
+hive.jdbc.url=jdbc:hive2://localhost:10000/
+
```

Comment:
Can you help add a comment and give an example to introduce Kerberos authentication for hive?

Reply:
In this PR, I didn't add Hive Kerberos auth.